### ISSUE TITLE
scp can't find private key

### ISSUE THREAD

**Issue by sbromberger** [blinksh/blink#761]
I've used `ssh` and `ssh2` to the host per some comments in earlier issues. I've also tried `[user]@snoopy` and that gave the same error.

```
blink> scp -vvv .ssh/known_hosts snoopy:linus-known-hosts
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0*   Trying 192.168.1.3...
* TCP_NODELAY set
* Connected to snoopy (192.168.1.3) port 22 (#0)
* SSH MD5 fingerprint: 1278[redacted]
* SSH host check: 0, key: AAAA[redacted]
* SSH authentication methods available: publickey,keyboard-interactive
* Using SSH private key file ''
* SSH public key authentication failed: Unable to extract public key from private key file: Unable to open private key file
* Failure connecting to agent
* Authentication failure
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
* Closing connection 0
curl: (67) Authentication failure
```

it looks like `config` doesn't write to `.ssh/id_rsa` or `.ssh/id_rsa.pub`.

Password authentication has been disabled on the server.

This is version 12.9 (108).

**Comment by Vinznx** [comment]
Same problem. Hope to get answer soon.

**Comment by carloscabanero** [comment]
These have all been fixed on 13.90. Closing here as we already have duplicates for SCP and SFTP issues.
